content/posts/newsletter-014/index.md

```diff
@@ -66,6 +66,100 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Library & Tooling Updates
 
+### [Bevy Engine][bevy]
+
+[![bevy logo](bevy_logo.png)](https://bevyengine.org/)
+
+[Bevy][bevy] is a refreshingly simple data-driven
```

Comment:
I like this intro, but I think that maybe we don't need such a big introduction in every month where Bevy is mentioned? 1-2 sentences would be ideal :)

Reply:
Sure thing!